File: models/medicine.go
package models

type Medicine struct {
	ID           uint    `json:"id" gorm:"primaryKey"`
	Name         string  `json:"name"`
	Description  string  `json:"description"`
	Category     string  `json:"category"`
	Dosage       string  `json:"dosage"`
	Manufacturer string  `json:"manufacturer"`
	Price        float64 `json:"price"`
	Availability bool    `json:"availability"`
}

File: models/Medicine.js
// models/Medicine.js
const mongoose = require('mongoose');

const MedicineSchema = new mongoose.Schema({
  name: { type: String, required: true },
  description: { type: String, required: true },
  category: { type: String, required: true },
  dosage: { type: String, required: true },
  manufacturer: { type: String, required: true },
  price: { type: Number, required: true },
  availability: { type: Boolean, required: true },
  image_url: { type: String } // Ссылка на изображение лекарства
});

module.exports = mongoose.model('Medicine', MedicineSchema);
